### DISCUSSION TITLE
The score has not yet been displayed on the EvalAI platform.

### DISCUSSION THREAD

**Discussion by TomatoPicker278** [empriselab/RCareWorld#164] [Other Q&A] [answered]
Hello, I submitted a simple version of a zip file as a team on EvalAI on October 31, but it’s still in the "submitted" status with no score displayed (as shown in the image). 
![截屏2024-11-04 12 29 47](https://github.com/user-attachments/assets/6e5754ad-7477-4e94-844c-87ed0f44c314)

I’m not sure if there was an issue with my submission process or if it’s a problem with the EvalAI testing system. If possible, I would like to communicate with the organizers and ask if they could help me check the specific reason for this. Thank you!

**Comment by YoruCathy** [comment]
@alkdischen will run your submission file soon.

**Reply by TomatoPicker278** [reply]
Thank you for your response. My zip file was submitted on 10/31/24 at 04:35:43 PM, so perhaps you can identify my file based on this timestamp. Thank you!

**Reply by shxingch** [reply]
Hi @TomatoPicker278 - I've checked all submissions but couldn't find yours from that time period. Could you please resubmit your file again and make sure it's set to public?

**Reply by TomatoPicker278** [reply]
> Hi @TomatoPicker278 - I've checked all submissions but couldn't find yours from that time period. Could you please resubmit your file again and make sure it's set to public?

OK，I will check it and submit it again soon

**Reply by TomatoPicker278** [reply]
@alkdischen Oh, we might have forgotten about the time difference. I submitted the file on 10/31/24 at 04:35:43 PM London time, and there’s a 5-hour time difference between us, so that would be 10/31/24 at 11:35:43 AM your local time. Could you please check if there is a file submitted around that time?

If not, I just submitted the exact same file again on 11/05/24 at 11:16:37 AM London time. Could you please check if either of these files exists? Thank you!

**Reply by shxingch** [reply, marked as answer]
Hi @TomatoPicker278! Your previous submission didn't go through completely due to some platform issues.  After your new submission, I can now see all your files (including former submission files). I've manually reviewed your submission files and your scores are now visible on the leaderboard!

**Reply by TomatoPicker278** [reply]
thanks!
